### PR TITLE
Quick footer year fix

### PR DESCRIPTION
**Pull request by OleanjiKingCode**
# Automated footer year

Before 
image

After

image

## Linked issues

 closes https://github.com/EveripediaNetwork/issues/issues/950

### CODE REVIEW

File: src/components/Layout/Footer/Footer.tsx [hunk 1]
```diff
@@ -78,7 +78,10 @@ const Footer = () => {
         <SimpleGrid fontSize="sm" columns={{ base: 1, lg: 2 }}>
           <Stack align={{ base: 'center', lg: 'flex-start' }} flex="1">
             <Flex alignItems="center">
-              <Text py={3} pr={2}>{`${t('copyRight')}`}</Text>
+              <Text py={3} pr={2}>
+                {' '}
+                © {new Date().getFullYear()} {`${t('copyRight')}`}
```
Comment (Adebesin-Cell) [inline src/components/Layout/Footer/Footer.tsx:83]: I think you can store this in a const.

**Review by Adebesin-Cell — APPROVED** [review]
Awesome

File: src/components/Layout/Footer/Footer.tsx (hunk 1 above)
Comment (OleanjiKingCode) [reply src/components/Layout/Footer/Footer.tsx:83]: Not sure it's needed in a var since it's only here we're using it. I'll do it tho

Comment (kesar) [reply src/components/Layout/Footer/Footer.tsx:83]: yup prob. not big deal

**Pull request merged**